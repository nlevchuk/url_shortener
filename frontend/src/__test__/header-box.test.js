import HeaderBox from '../header-box';

describe('<HeaderBox />', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<HeaderBox />);
    expect(wrapper).toMatchSnapshot();
  });
});
