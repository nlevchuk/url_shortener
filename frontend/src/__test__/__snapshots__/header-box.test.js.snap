// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HeaderBox /> should render correctly 1`] = `
<header>
  <div
    className="angle"
    id="left-angle"
  />
  <div
    id="logo"
  >
    Url shortener
  </div>
  <div
    className="angle"
    id="right-angle"
  />
</header>
`;
